ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -0,0 +1,301 @@
+import * as React from 'react';
+
+import {Component} from '@angular/core';
+
+import {Button, Clickable} from 'app/components/buttons';
+import {FlexColumn, FlexRow} from 'app/components/flex';
+import {ClrIcon} from 'app/components/icons';
+import {TextInput} from 'app/components/inputs';
+import {workspaceAdminApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {ReactWrapperBase} from 'app/utils';
+import {
+  getSelectedPopulations,
+  getSelectedResearchPurposeItems
+} from 'app/utils/research-purpose';
+
+import {
+  AdminWorkspaceResources,
+  UserRole,
+  Workspace
+} from 'generated/fetch';
+
+
+const styles = {
```

Comment:
nit: best practice is to use reactStyles({ ... }) for better type-safety. See [code](https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/index.tsx#L207) for details.